src/app/(routes)/across-plus/_components/get-started-section-top.tsx

```diff
@@ -0,0 +1,34 @@
+import Image from "next/image";
+import Autoplay from "embla-carousel-autoplay";
+
+import { GetStartedSection } from "@/app/_components/get-started-section";
+import { Carousel, CarouselContent, CarouselItem } from "@/app/_components/carousel";
+import { INTEGRATION_LINKS } from "@/app/_constants/links";
+import plusGetStartedImage from "@/app/_assets/plus-get-started.png";
+import mockupImage1 from "../_assets/mockup-1.png";
+import mockupImage2 from "../_assets/mockup-2.png";
+import mockupImage3 from "../_assets/mockup-3.png";
+import mockupImage4 from "../_assets/mockup-4.png";
```

Comment:
Should we use @/app here?

Reply:
These assets are not directly under `src/app/_assets` but rather in `src/app/(routes)/across-plus/_assets`. That's why I prefer `../` here because it's shorter

Reply:
Ah gotcha - sgtm